addons/account/views/report_configurator.xml

```diff
@@ -30,7 +30,19 @@
                         </t>   
                     </select>
                 </div>
-                <div class="col-sm-3">
+		<div class="col-sm-3" t-if="report.get_report_type() == 'bank_reconciliation'">
```

Comment:
Maybe we should create a new model that would be used to manage all those kinds of filters